core/domain/services/admin/privacy/policy/privacy_policy.template.php

```diff
@@ -0,0 +1,72 @@
+<h2><?php esc_html_e('Event Registration Data', 'event_espresso');?></h2>
```

Comment:
plz document all variables imported into the template at the top of the file as you would for any phpdoc

Reply:
good idea